src/lib/math/bigint/big_code.cpp

```diff
@@ -85,7 +85,7 @@ std::string BigInt::to_dec_string() const
    return s;
    }
 
-std::string BigInt::to_hex_string() const
+std::string BigInt::to_hex_string(const bool with_hex_prefix /*= true*/) const
```

Comment:
The `= true` comment is not really needed here. Good to know during code review, as it communicates that the old behaviour is still the default, though. :+1:

```suggestion
std::string BigInt::to_hex_string(const bool with_hex_prefix) const
```